data/language/en-GB.txt

```diff
@@ -4232,6 +4232,9 @@ STR_5920    :Render weather effects
 STR_5921    :{SMALLFONT}{BLACK}If enabled, rain and gloomy colours will be rendered during storms.
 STR_5922    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{SMALLFONT}{BLACK}Max {STRINGID}
 STR_5923    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{SMALLFONT}{BLACK}Max {COMMA16} {STRINGID} per train
+STR_5924    :New folder
```

Comment:
Command line tools versus graphical user interface. I'm not disputing the use of 'directory' internally -- it is the technical term for it, no doubt.

The file / folder analogy is used consistently in GUIs though -- not just in strings, but also in icons. I do believe we should follow that -- the consistency argument works that way, too.